x-pack/plugins/security_solution/public/management/pages/policy/view/components/config_form/index.stories.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { storiesOf, addDecorator } from '@storybook/react';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import { EuiCheckbox, EuiSpacer, EuiSwitch, EuiText } from '@elastic/eui';
+
+import { ConfigForm } from '.';
+
+addDecorator((storyFn) => (
+  <ThemeProvider theme={() => ({ eui: euiLightVars, darkMode: false })}>{storyFn()}</ThemeProvider>
+));
+
+storiesOf('PolicyDetails/ConfigForm', module)
+  .add('One OS', () => {
+    return (
+      <ConfigForm type="Type 1" supportedOss={['windows']}>
```

Comment:
should we use the os enums you made here?

Reply:
If you refer to os.ts file then it contains only type definitions, unfortunately literal types can not be used as keys/symbols directly. I plan to later revisit this and add some constant to keep the keys.